src/resource-graph/azext_resourcegraph/_help.py

```diff
@@ -0,0 +1,38 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+# pylint: disable=line-too-long
+
+helps['graph query'] = """
+    type: command
+    short-summary: Query the resources managed by Azure Resource Manager.
+    long-summary: >
+        Learn more about the query syntax here: https://aka.ms/graph/learntoquery
```

Comment:
I tried to visit this link, but it didn't work for me.

Reply:
Good catch! Replaced the link.